component/src/governance/execute.rs

```diff
@@ -66,6 +69,12 @@ pub async fn proposal_submit(
         .await;
     state.put_proposal_voting_end(proposal_id, voting_end).await;
 
+    // If there was a proposal submitted, ensure we track this so that clients
+    // can retain state needed to vote as delegators
+    let mut compact_block = state.stub_compact_block();
+    compact_block.proposal_started = true;
+    state.stub_put_compact_block(compact_block);
+
```

Comment:
this moved out of the `Transaction` actionhandler, which now just calls into the options